praw/models/reddit/subreddit.py

```diff
@@ -6,10 +6,11 @@
 from urllib.parse import urljoin
 
 from prawcore import Redirect
+import socket as socket_module
```

Comment:
Rather than renaming the import, can we rename the variable to use `websocket`?

Reply:
Using `connection`, as `websocket` is a different module imported in this file.